deploy/helm/flightctl/charts/ui/templates/flightctl-ui-cm.yaml

```
@@ -7,6 +7,7 @@ metadata:
     {{- include "flightctl.standardLabels" . | nindent 4 }}
 data:
   FLIGHTCTL_SERVER: {{ .Values.api.url | quote }}
+  FLIGHTCTL_EXTERNAL_URL: {{ include "flightctl.getApiUrl" . }}
```

I'd rename so it's clear that it's the same as the FLIGHTCTL_SERVER, just external.

```suggestion
  FLIGHTCTL_SERVER_EXTERNAL: {{ include "flightctl.getApiUrl" . }}
```